src/streams/receive.rs

```diff
@@ -524,7 +524,7 @@ impl StreamRx {
     }
 
     fn nack_enabled(&self) -> bool {
-        self.rtx.is_some() && !self.suppress_nack
+        !self.suppress_nack
```

Comment:
This is not correct, because it would enable nack also for streams that we don't want nack for.

Reply:
audio ?